README.md

```diff
@@ -2,24 +2,173 @@
+{
+	var corsPolicyName = "Some Cors policy blah blah";
+    app.UseJsonExceptionPages(corsPolicyName)
+       .UseStaticFiles() // etc.
+       .UseMvc();
+}
+```
+
+4. Some custom exception handling, during the exception. 
+
+This would be if you want to return some specific result for a specific status code. For example, if the status code was an HTTP409 you might have a custom json schema. While for everything else, you have some other result.
+
+```
+public void ConfigureServices(IServiceCollection services)
+{
+    // Common/standard usage: some nice consistent and readable Json settings.
+    services.AddACommonJsonFormatter();
+}
+
+public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+{
+    JsonExceptionPageResult HandleException(Exception result) 
+    { 
+        // E.g. Real life scenario:
+        //      If status code == 401, then return some result
+        //      else if status code == 409 then resturn another result
+        //      else return some default result or return null (for default handling to kick in).
+
+        // Contrite hardcoded example to return a hardcoded result.
+        // Of course you wouldn't *really* do this, in production.
+        return new JsonExceptionPageResult
+        {
+            StatusCode = System.Net.HttpStatusCode.UpgradeRequired,
+            ApiErrors = new List<ApiError>
+            {
+                new ApiError
+                {
+                    Message = "I'm a little tea pot, short and stout."
+                }
+            }
+        };
+    };
+
+    app.UseJsonExceptionPages(customExceptionFunction: HandleException)
+       .UseStaticFiles() // etc.
+       .UseMvc();
+}
+```
+
+### <a name="Sample3">Automatic Model Validation via FluentValidation.</a>
+
+```
+public void ConfigureServices(IServiceCollection services)
+{
+    // Reflect through the current assembly looking for FluentValidation Validators 
+    services.AddCustomFluentValidation(this.GetType());
+}
+
+ -- or --
+
+public void ConfigureServices(IServiceCollection services)
+{
+    // Reflect through the *all* assemblies looking for any FluentValidation Validators. 
+    var types = new [] { typeof(Startup), typeof(AnotherClassFromAnotherAssembly) };
+    services.AddCustomFluentValidation(types);
+}
+```
 
 ---
 
 ## Contributing
 
-Discussions and pull requests are encouraged :) Please ask all general questions in this repo or pick a specialized repo for specific, targetted issues. We also have a [contributing](https://github.com/OpenRealEstate/OpenRealEstate/blob/master/CONTRIBUTING.md) document which goes into detail about how to do this.
+Discussions and pull requests are encouraged :) Please ask all general questions in this repo or pick a specialized repo for specific, targetted issues. We also have a [contributing](https://github.com/OpenRealEstate/Homely/blob/master/CONTRIBUTING.md) document which goes into detail about how to do this.
```

Comment:
update URL

Reply:
AH! that was offscreen, which was how I missed it :(